rails_event_store_active_record/lib/rails_event_store_active_record/event_repository.rb

```diff
@@ -39,27 +39,54 @@ def last_stream_event(stream_name)
     end
 
     def read_events_forward(stream_name, after_event_id, count)
-      @repo_reader.read_events_forward(stream_name, after_event_id, count)
+      RubyEventStore::Specification.new(self)
+        .stream(stream_name)
+        .from(after_event_id)
+        .limit(count)
+        .each
+        .to_a
```

Comment:
why is this `to_a` needed?

Reply:
Reminder that in that phase public API is not yet changed and the expectation is to get an array.  This fulfills this expectation.

Reply:
We'll change it when client (specs it that case) call `to_a`.